ports/opendnp3/portfile.cmake

```
@@ -27,11 +26,8 @@ vcpkg_install_cmake()
 
 vcpkg_copy_pdbs()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH share/asiodnp3 TARGET_PATH share/asiodnp3)
-vcpkg_fixup_cmake_targets(CONFIG_PATH share/asiopal TARGET_PATH share/asiopal)
-vcpkg_fixup_cmake_targets(CONFIG_PATH share/opendnp3 TARGET_PATH share/opendnp3)
-vcpkg_fixup_cmake_targets(CONFIG_PATH share/openpal TARGET_PATH share/openpal)
-
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake TARGET_PATH share/${PORT})
```

```suggestion
vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake TARGET_PATH share/asio)
```
It would be better to use the name here.

---

should have been opendnp3 instead of asio